docker/tinyms/0.1.0/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG BASE_CONTAINER=mindspore/mindspore-cpu:1.1.1
+FROM $BASE_CONTAINER
+
+LABEL MAINTAINER="Leon Wang <wanghui71leon@gmail.com>"
+
+# Install base tools
+RUN apt update
```

Comment:
apt get update ?

Reply:
In Ubuntu 18.04 version, `apt` equals with `apt-get`.